felles/src/main/kotlin/no/nav/helsearbeidsgiver/felles/Resultat.kt

```diff
@@ -15,18 +13,11 @@ data class TilgangResultat(
 )
 
 @Serializable
-data class TrengerData(
+data class HentForespoerselResultat(
+    val sykmeldtNavn: String,
+    val avsenderNavn: String,
```

Comment:
rename denne til innsenderNavn kanskje?  - så blir det en mindre "oversettelse" i HentForespoerselResultat.toResponse()

Reply:
Jeg spår `HentForespoerselResponse` en snarlig død, med en påfølgende gjenfødsel fra asken i ny og bedre form. Så jeg lar forskjellen fare for nå.